docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -556,45 +555,15 @@ and geography, we can calculate the probability of moving from ACS, as
 the weighted average of MIGPUM.isnull(); could also determine if they
 moved within the PUMAs represented in the sim or from outside those
 PUMAs.
+For now, we only model migration within the sim catchment area (this component)
+and to/from other countries (next two components).
+When the simulation only includes part of the US, there is no domestic
+migration into or out of this region.
```

Comment:
just to make sure i'm reading this correctly:

we're saying that when we are only modeling a subset of the US, there is no migration from the subset we are modeling to another part of the US, or migration from another part of the US to the subset we are modeling -- yes?

Reply:
Yes, that's correct.